### PR TITLE
Add support for filtering query string parameters

### PR DESCRIPTION
**Pull request by joelcrocker**
This is just a separate PR/fork for attempting to fix CI test issues on https://github.com/bugsnag-elixir/plugsnag/pull/51

### CODE REVIEW

**Comment by joelcrocker** [comment]
@nirev mind approving Github workflows for me so I can work on getting CI passing? Thanks!

**Comment by nirev** [comment]
I think this is good to go, the issue with the CI on the other one were bumped deps in mix.lock that were incompatible with older elixir versions.

File: README.md [hunk 1]
```diff
@@ -53,6 +53,15 @@ config :plugsnag, filter: [params: ~w(password password_confirmation super_sekri
 
 ```
 
+By default, query strings are not filtered and may still leak sensitive information stored there
+(which shouldn't be, anyway). To filter the query string in the generated report, set the
+`:filter_query_string` config option to true:
+
+```elixir
+config :plugsang, filter_query_string: true
```
Comment (nirev) [inline README.md]:
```suggestion
config :plugsnag, filter_query_string: true
```

**Comment by joelcrocker** [comment]
@nirev thank you so much for the quick review, merge, and release-cutting! It's appreciated!

**Pull request merged**